ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -60,7 +60,8 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
 
     // Associate the connection details with the session
     let session;
-    const { session_id, address, port, credentials } = connectionDetails;
+    const { session_id, address, port, credentials, expiration_time } =
```

Comment:
The expiration field returned from a connection is actually called `expiration`

Reply:
@ZedL can you elaborate a little further, so when we destructor those details in this file, are we getting those details from the `connect` method in the `ipc`.  In there the field is called `expiration_time`

Reply:
Can you try connecting and seeing what JSON you get back from the binary?

Reply:
I just looked into what the binary returns from the `connect` command and it is `expiration`. We will have to update the destructor here and assign it as `expiration_time: expiration` for the push to the store.

Reply:
That is correct, the binary returns `expiration` object. This should mimic that behaviour.